spec.html

```diff
@@ -26540,7 +26541,7 @@ <h1>Properties of the Symbol Constructor</h1>
 
       <emu-clause id="sec-symbol.asynciterator">
         <h1>Symbol.asyncIterator</h1>
-        <p>The initial value of `Symbol.asyncIterator` is the well known symbol @@asyncIterator (<emu-xref href="#table-1"></emu-xref>).</p>
+        <p>The initial value of `Symbol.asyncIterator` is the well known symbol %Symbol.asyncIterator% (<emu-xref href="#table-1"></emu-xref>).</p>
```

Comment:
These should have `<dfn>` on them, otherwise auto‑linking won’t work:
```suggestion
        <p>The initial value of `Symbol.asyncIterator` is the well known symbol <dfn>%Symbol.asyncIterator%</dfn> (<emu-xref href="#table-1"></emu-xref>).</p>
```

Reply:
It will work without dfns once ecmarkup is modified to accept the new syntax, which it will be before merging this PR.

Reply:
**ecmarkup** already supports this syntax, since `%TypedArray.prototype%` links to <https://tc39.es/ecma262/#sec-properties-of-the-%typedarrayprototype%-object>.

Reply:
In that case i think intrinsics should all not require the dfn, unless all the rest of them have it?

Reply:
All the rest of the intrinsics have `<dfn>`s, as it’s necessary for auto‑linking.

Reply:
fair enough, then these should too, thanks :-)

Reply:
<https://github.com/tc39/ecma262/pull/1996> has now landed, so this needs to be rebased on top of it.